solutions/bio2/ScoopOne/index.tsx

```diff
@@ -0,0 +1,60 @@
+        </SText>
+        <SText>
+          The package is at $j$ locked with a lock other than $j$, and lock $j$ gets added
+        </SText>
+        <SText>
+          The package is at $i$ locked with lock $i$ and $j$, and lock $i$ gets removed.
+        </SText>
+        <SText>
+          The package is at $j$ locked with lock $j$, and lock $j$ gets removed.
+        </SText>
+      </SList>
+      <SText>
+        At this point it is evident that there is no need to use any lock other than locks $i$ and $j$. Between events $1$ and $2$ if another lock is added, it must be removed before event $2$, lock $i$ will need to be removed early, and the other lock will also need to be removed later. This is unnecessary and suboptimal. Between events $2$ and $3$ there is no room to add another lock. Between events $3$ and $4$ there is no need to add a lock, as the package is locked correctly with $j$. Thus the minimum total distance travelled is $d(i, j) + d(j, i) + d(i, j) = 3 d(i, j)$.
+      </SText>
+      
+      <STitle>Reduction to a simpler problem</STitle>
+      <SText>
+        Let us name the messenger we have been considering so far "Rob". At this point, consider another messenger, Bob, traversing the village who can be trusted not to look at his package, so no locks are needed. We have shown that the Rob will take exactly $3$ times long as Bob to deliver the package to the villages in the same order. Thus the answer for Rob is $3$ times the answer for Bob, i.e. $3$ times the length of the shortest walk on the tree that visits every vertex. 
+      </SText>
+      <STitle>Solving the simpler problem </STitle>
+      
+      <HStack>
+      <SText>
+        We now proceed to solve this simpler problem. Suppose that Bob lives in the starting node, node $i$. After taking the package through every node he returns home. Clearly Bob must walk on every edge of the graph, otherwise he will miss a node. Furthermore, Bob must walk on every edge of the graph at least twice, as if he walks down edge $p$ to $q$ only once, he will be stuck in the subtree of node $q$ and not able to return home. Moreover if Bob walks in the order of a DFS he can achieve this lower bound, walking on each edge exactly twice and returning home. Thus Bob can travel the minimum distance of $2E$. We have $\min$(distance to reach all vertices + distance from last vertex to home) $ = 2E$. So $\min$(distance to reach all vertices) = 2E - $\max$(distance from last vertex to home). Thus the answer to the simpler problem is $2E - h$ where $h$ is the height of the tree. 
+      </SText>
+      <Image src="/assets/images/scoopone/scoopexample.png" height="16rem" alt="Sample Diagram" />
+      </HStack>
+      <STitle>Solution </STitle>
+      <SText>
+        Combining the above we get that the answer is $6E - 3h$. $E$ is simply one less than $v$, as the graph is a tree. $h$ can be computed with a DFS in $\mathcal{brace("O")}(n)$.
```

Comment:
Sorry, I missed this on my first review but I think it's better not to start a sentence with a latex symbol. 

The $E$ debatably doesn't look weird since it's a capital letter (if you can change it, this would be great though) but you should definitely change " $h$ can be" to maybe like - "The value of $h$ can be ..."